docs/src/ci.md

```diff
@@ -123,16 +121,18 @@ Suggested configuration
    and crash. Learn more about this option in [Docker docs](https://docs.docker.com/engine/reference/run/#ipc-settings---ipc).
 1. Seeing other weird errors when launching Chromium? Try running your container
    with `docker run --cap-add=SYS_ADMIN` when developing locally.
-1. [dumb-init](https://github.com/Yelp/dumb-init) is worth checking out if you're
-   experiencing a lot of zombies Chromium processes sticking around. There's special
-   treatment for processes with PID=1, which makes it hard to terminate Chromium
-   properly in some cases (e.g. in Docker).
+1. Using `--init` Docker flag or [dumb-init](https://github.com/Yelp/dumb-init) is recommended to avoid special
+   treatment for processes with PID=1. This is a common reason for zombie processes.
 
 ### Azure Pipelines
 
 For Windows or macOS agents, no additional configuration required, just install Playwright and run your tests.
 
-For Linux agents, you can use [our Docker container](./docker.md) with Azure Pipelines support for [running containerized jobs](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/container-phases?view=azure-devops). Alternatively, you can refer to the [Dockerfile](./docker.md) to see additional dependencies that need to be installed on a Ubuntu agent.
+For Linux agents, you can use [our Docker container](./docker.md) with Azure
+Pipelines support for [running containerized
+jobs](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/container-phases?view=azure-devops).
+Alternatively, you can use [Command line tools](./cli.md#install-system-dependencies) to install all necessary dependencies for Ubuntu 18.04
```

Comment:
Let's avoid `Ubuntu 18.04` because it will become stale. Just say "necessary dependencies"?